TorchSharp/TorchSharp.csproj

```diff
@@ -44,9 +45,18 @@
     <Compile Remove="T4Template.cs" />
     <Compile Remove="Handle.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="System.Numerics.Tensors" Version="0.1.0-preview2-181029-2" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.5.2" />
```

Comment:
`4.5.2` makes me nervous. Did we just get a dependency on .NET Framework instead of .NET Core / Standard?

Reply:
Mmm I don't know. I got this from SNT. Looking at [this](https://www.nuget.org/packages/System.Runtime.CompilerServices.Unsafe/) it seems that the only dependency is on Runtime which is Standard 1.0. Does this make sense?